Advance Character Search/localization/english/acs_l_simp_chinese.yml

```diff
@@ -0,0 +1,62 @@
+l_simp_chinese:
+ acs_sort_by_diplomacy:0 "按 [diplomacy|E] 排序"
+ acs_sort_by_martial:0 "按 [martial|E] 排序"
+ acs_sort_by_stewardship:0 "按 [stewardship|E] 排序"
+ acs_sort_by_intrigue:0 "按 [intrigue|E] 排序"
+ acs_sort_by_learning:0 "按 [learning|E] 排序"
+ acs_sort_by_prowess:0 "按 [prowess|E] 排序"
+ acs_claim_tooltip:0 "#T 拥有 [claims|E] 头衔的宣称：#!\n[CharacterListItem.ListTitleItems]"
+ acs_sort_by_magic_sauce:0 "能力总和（考虑健康和年龄因素）"
+ not_ruler_and_not_heir_filter_option:0 "非统治者或继承人"
+ independent_filter_option:0 "独立统治者"
+ younger_adult_filter_option:0 "青年人（16~34岁）"
+ older_adult_filter_option:0 "老年人（35岁以上）"
+ is_betrothed_filter:0 "已订婚"
+ not_betrothed_filter:0 "未订婚"
+ know_secret_filter:0 "有秘密（测试了好些次，实在不是明白具体作用）"
```

Comment:
Maybe we have a bug with this function. It should be the function which is checking if playing character know any secrets about that other character.